scrapy/commands/parse.py

```diff
@@ -7,7 +10,9 @@
 from scrapy.utils.conf import arglist_to_dict
 from scrapy.utils.spider import iterate_spider_output, spidercls_for_request
 from scrapy.exceptions import UsageError
-from scrapy import log
+
+logger = logging.getLogger('scrapy')
```

Comment:
I like the idea. Actually I used `scrapy` logger for a kind of weak reason, I wanted to maintain the `[scrapy]` prefix on the logging format and that's the easiest way (just need to provide a *[%(name)s]* on the logformatter). I think I prefer to name each logger individually (for example, I want to have a `scrapy.requests` logger to mimic Django's `django.requests`) but it's probably better to use `__name__` as default on each module and treat those cases separately.